apps/frontend/src/libs/components/popover/popover.tsx

```diff
@@ -2,36 +2,43 @@ import {
 	useCallback,
 	useHandleClickOutside,
 	useRef,
-	useState,
 } from "~/libs/hooks/hooks.js";
 
 import styles from "./styles.module.css";
 
 type Properties = {
 	children: React.ReactNode;
 	content: React.ReactNode;
+	isPopoverOpened: boolean;
+	onPopoverClose: () => void;
+	onPopoverOpen: () => void;
```

Comment:
This looks like we violating A5 quality criteria. Also names `onClose` and `onOpen` don't represent what they actually do: `onOpen` used to open popover, and `onClose` is used to close it -- so they are not handlers but rather actions. 
So according to all that I said:
```suggestion
	isOpened: boolean;
	close: () => void;
	open: () => void;
```
Also it would be nice to do the same changes for modal component

Reply:
agree, but instead close/open should be, onClose, onOpen